src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -224,6 +231,18 @@ private void ProcessMarkdownInfo(MarkdownInfo markdownInfo)
         /// </summary>
         protected override void EndProcessing()
         {
+            if (_inputObjectBuffer.Count > 0)
+            {
+                if (_inputObjectBuffer.Count == 1)
+                {
+                    ConvertFromMarkdown("InputObject", _inputObjectBuffer[0]);
+                }
+                else
+                {
+                    ConvertFromMarkdown("InputObject", string.Join(Environment.NewLine, _inputObjectBuffer.ToArray()));
+                }
+            }
```

Comment:
Similar code copied from `ConvertFrom-Json` to avoid allocations if just one string exists.

Reply:
Removed this as I don't see benefit of having it in terms of performance.

Reply:
Converted this into a ternary as it seems there is a perf improvement for not calling `string.Join` on a single string.